Sources/Signal.swift

```diff
@@ -1847,12 +1851,18 @@ extension SignalProtocol {
 			return disposable
 		}
 	}
-
-	/// Debounce values sent by the receiver, such that at least `interval`
-	/// seconds pass after the receiver has last sent a value, then forward the
-	/// latest value on the given scheduler.
+	
+	/// Don't send a value until at least `interval` seconds have passed since
```

Comment:
Similar change to above:

```
/// Returns a signal that sends values only when `interval` seconds have elapsed 
/// since this signal has last sent a value.
///
/// If this signal always sends values more frequently than `interval` seconds, then the 
/// returned signal will never send any values. 
///
/// - seealso: `throttle`
```

If we ever get the Jazzy stuff online in html form, then we can specify a URL with the `seealso`.